pkg/model/context.go

```diff
@@ -143,6 +143,12 @@ func (m *KopsModelContext) CloudTagsForInstanceGroup(ig *kops.InstanceGroup) (ma
 		labels[k] = v
 	}
 
+	// Apply NTH Labels
+	nth := m.Cluster.Spec.NodeTerminationHandler
+	if nth != nil && fi.BoolValue(nth.Enabled) && fi.BoolValue(nth.EnableSqsTerminationDraining) {
+		labels["aws-node-termination-handler/managed"] = "true"
```

Comment:
nit: the official docs use an empty string for the tag value. the code [doesn't check the value at all](https://github.com/aws/aws-node-termination-handler/blob/4a370232d42826a7081cee9d90a80acb9969815c/pkg/monitor/sqsevent/sqs-monitor.go#L236-L240).

```suggestion
		labels["aws-node-termination-handler/managed"] = ""
```

Reply:
👍 I'll remove the value in the next revision